vllm/utils.py

```diff
@@ -1095,9 +1095,9 @@ def cuda_device_count_stateless() -> int:
 
 
 #From: https://stackoverflow.com/a/4104188/2749989
-def run_once(f):
+def run_once(f: Callable[P, None]) -> Callable[P, None]:
```

Comment:
shouldn't this return Any techncially?

Reply:
Technically yes, but I believe it's designed for functions that return `None` only. (Otherwise, it would be confusing for the function to sometimes return the original type and other times `None`)